quantuminspire/util/configuration.py

```diff
@@ -39,7 +39,7 @@ def json_config_settings(settings: BaseSettings) -> Any:
 class Settings(BaseSettings):  # pylint: disable=too-few-public-methods
     """The settings class for the Quantum Inspire persistent configuration."""
 
-    auths: str = "auths"
```

Comment:
`auths` shouldn't be removed. The structure I want to go for is the same as with `config.json` for Docker. It has an object containg all hostnames you have credentials for. In this case, the  temporary `user_id` should be part of these hostname settings; e.g.

```json
"auths": {
  "staging.quantum-inspire.com": {
    "user_id": 1
  },
  "qa.quantum-inspire.com": {
    "user_id": 2
  },
  "production.quantum-inspire.com": {
    "user_id": 3
  }
}
```

Reply:
Could you check the format I added?